### PR TITLE
embulk 0.10.49

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ZhongRuoyu** [comment]
Test timed out.

**Review by chenrui333 — DISMISSED** [review]

File: Formula/embulk.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Embulk < Formula
   # https://www.embulk.org/articles/2020/03/13/embulk-v0.10.html
   # v0.10.* is a "development" series, not for your production use.
   # In your production, keep using v0.9.* stable series.
```
Comment (ZhongRuoyu) [inline Formula/embulk.rb:6]: Mmm, actually, there must be something wrong.

Comment (chenrui333) [reply Formula/embulk.rb:6]: exactly, I created [an issue](https://github.com/embulk/embulk/issues/1613) for discussing whether this upgrade is really needed.

**Comment by chenrui333** [comment]
For now, let's close this issue for now. (we can reopen this later)

**Comment by dmikurube** [comment]
See: https://www.embulk.org/articles/2023/04/13/embulk-v0.11-is-coming-soon.html

btw, as of now, you may want to upgrade to v0.9.25, instead. https://github.com/embulk/embulk/releases/tag/v0.9.25

**Comment by ZhongRuoyu** [comment]
Thanks @dmikurube. We will ship v0.9.25 for now: #129043

**Pull request closed without merging**